### DISCUSSION TITLE
Why prisma size (mb) is so large?

### DISCUSSION THREAD

**Discussion by jvitormelo** [prisma/prisma#24272] [Q&A]
### Question

When I deployed my functions to serverless, with drizzleORM the size was less than 1mb
![image](https://github.com/prisma/prisma/assets/66218248/00d90ab2-5d2a-4cfd-ad16-e2abdcaac65a)

If I do the same with prismaORM the size is so much large
![image](https://github.com/prisma/prisma/assets/66218248/0a2337e0-c28f-4b22-bb4c-0c01218484ca)

**Comment by ludralph** [comment]
Hi @jvitormelo 👋 

Thank you for raising this question.

The Prisma query engine is a binary that is included in your deployment. This engine is responsible for translating your Prisma queries into SQL queries that your database can execute. The inclusion of this binary significantly increases the deployment size that's why you're seeing a large file from Prisma.
